src/test/java/com/google/cloud/spanner/jdbc/JdbcDatabaseMetaDataWithMockedServerTest.java

```diff
@@ -306,4 +331,97 @@ public void getSchemasInDdlBatch() throws SQLException {
       connection.createStatement().execute("ABORT BATCH");
     }
   }
+
+  @Test
+  public void verifyGoogleSql() throws SQLException {
```

Comment:
method name is `verifyGoogleSql()`, but we seem to be validating the true assertions in the case `dialect == Dialect.POSTGRESQL`. Should the method name be different?

Reply:
Yeah, the method name is a little confusing. What we are verifying here is that the `/*GSQL*/` header is kept in the SQL statement without comments if the dialect is PostgreSQL, and that it is removed if the dialect is Google_Standard_Sql.
I've added a comment to the test and updated the test name to `verifyGoogleSqlHeaderIsCorrectlyParsed`.